components/form/CheckboxGroup.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { CheckboxLabel, ErrorLabel } from '.';
+
+export function Checkbox({ ...props }) {
```

Comment:
This doesn't make have any effect as you could have written this as `Checkbox(props)`; it is better to destructure the needed variables directly.

```suggestion
export function Checkbox({ label, name, value }) {
```